### PR TITLE
Add spec ipsec_conf

### PR DESCRIPTION
**Pull request by shlao**
Added
```
insights/parsers/ipsec_conf.py
insights/parsers/tests/test_ipsec_conf.py
docs/shared_parsers_catalog/ipsec_conf.rst
```

Updated
```
insights/specs/__init__.py
insights/specs/default.py
```
Signed-off-by: shlao <shlao@redhat.com>

### CODE REVIEW

File: insights/parsers/ipsec_conf.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        if not content:
+            raise SkipException('No content.')
+
+        ipsec_type, ipsec_name = "", ""
+        ipsec_sections = {}
+
+        for line in get_active_lines(content):
+            if line.startswith('include '):
+                fields = [field.strip() for field in line.split()]
+                if len(fields) != 2:
```
Comment (xiangce) [inline insights/parsers/ipsec_conf.py:67]: Is this an "Exception" that should be caught?

File: insights/parsers/ipsec_conf.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    continue
+
+                array = self.get('include', [])
+                array.append(fields[1])
+                self['include'] = array
+                continue
+
+            if line.startswith('conn ') or line.startswith('config '):
+                fields = [field.strip() for field in line.split()]
+                if len(fields) != 2:
```
Comment (xiangce) [inline insights/parsers/ipsec_conf.py:77]: Same comment as above

File: insights/parsers/ipsec_conf.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    continue
+
+                array = self.get('include', [])
+                array.append(fields[1])
+                self['include'] = array
+                continue
+
+            if line.startswith('conn ') or line.startswith('config '):
```
Comment (xiangce) [inline insights/parsers/ipsec_conf.py:75]: `if line.startswith(('conn ', 'config '))`

File: insights/specs/sos_archive.py [hunk 4]
```diff
@@ -220,6 +220,7 @@ class SosSpecs(Specs):
     sssd_logs = glob_file("var/log/sssd/*.log")
     samba_logs = glob_file("var/log/samba/log.*")
     ssh_foreman_config = simple_file("/usr/share/foreman/.ssh/ssh_config")
+    subscription_manager_id = simple_file("/sos_commands/subscription_manager/subscription-manager_identity")
```
Comment (bfahr) [inline insights/specs/sos_archive.py:223]: @shlao why is this change in your PR?

Comment (bfahr) [reply insights/specs/sos_archive.py:223]: You probably need to rebase your PR with master.

**Review by xiangce — APPROVED** [review]

**Comment by xiangce** [comment]
@shlao - The last several updates look good. I'll go ahead and merge this PR.
But please do not use "--amend" and "--force" in the following to push the new commits to the PR, which will cover the former commits and make it difficult for reviewers to check the changes. Thanks.

**Pull request merged**